test/unit/lib/runner/writers/index-tests.js

```diff
@@ -34,4 +35,11 @@ describe('lib/runner/writers/index.js', function() {
 
     expect(result.JSON).to.deep.equal(json);
   });
+
+  it('should have the property \'actionJSON\' with the value from requiring'
+    + ' \'./action-json-writer.js\'', function() {
+    let result = require('../../../../../lib/runner/writers');
+
+    expect(result.actionJSON).to.deep.equal(json);
```

Comment:
this should not equal json, it needs to make sure its NOT json, a new sinon stub should be created to make sure it returns uniquely for both json and actionjson